src/urg_stamped.cpp

```diff
@@ -483,6 +517,7 @@ class UrgStampedNode
     , timestamp_hpf_(20)
     , timestamp_outlier_removal_(ros::Duration(0.001), ros::Duration())
     , timestamp_moving_average_(5, ros::Duration())
+    , error_count_{ 0, 0 }
```

Comment:
Removing from here and adding a constructor to `ResponseErrorCount` would be better.

Reply:
done